src/Rule/Rule.php

```diff
@@ -117,13 +117,21 @@ public static function parse(ParserState $oParserState): Rule
      * @param string $sRule
      *
      * @return array<int, string>
+     * The first item is the innermost separator (or, put another way, the highest-precedence operator).
+     * The sequence continues to the outermost separator (or lowest-precedence operator).
      */
     private static function listDelimiterForRule($sRule): array
     {
         if (\preg_match('/^font($|-)/', $sRule)) {
             return [',', '/', ' '];
         }
-        return [',', ' ', '/'];
+
+        switch ($sRule) {
+            case 'src':
+                return [' ', ','];
+            default:
+                return [',', ' ', '/'];
```

Comment:
Maybe we can integrate the preg matching into the `switch` as `switch` also allows general conditions: https://www.php.net/manual/en/control-structures.switch.php

Reply:
I don't see how.  Doesn't using general conditions require opening with `switch (true)`?

Reply:
Hm… or maybe we can create a data structure "regexp => delimiters" for this?

Reply:
> Hm… or maybe we can create a data structure "regexp => delimiters" for this?

I'd suggest leaving this until more cases have been added, then we'd have a clearer idea about what would be best.